### ISSUE TITLE
Entry Editor: Add tab for special fields

### ISSUE THREAD

**Issue by simonharrer** [JabRef/jabref#480]
At the moment, the special fields like rank etc. can only be edited in the main table, but not in the entry editor.

**Comment by stefan-kolb** [comment]
I don't think we need this now. If so we should first rework special fields in general.

**Comment by koppor** [comment]
Reworking is discussed at https://github.com/koppor/jabref/issues/84. Should we really do it? Would it be meanwhile OK to include an extra tab in the editor?

**Comment by simonharrer** [comment]
@stefan-kolb is this already possible in the tab containing all the fields which are not shown in the other tabs?

**Comment by stefan-kolb** [comment]
No? This tab just shows all missing fields but no special fields.

**Comment by stefan-kolb** [comment]
I don't like the idea because we first need to rework and redefinde the special fields in my opinion. Therefore, voting to close this issue for now.
